go/libkb/rpc_exim.go

```diff
@@ -1393,7 +1397,7 @@ func (e ChatInternalError) ToStatus() keybase1.Status {
 func (e ChatConvExistsError) ToStatus() keybase1.Status {
 	kv := keybase1.StringKVPair{
 		Key:   "ConvID",
-		Value: e.ConvID.String(),
+		Value: hex.EncodeToString(e.ConvID),
```

Comment:
It's more explicit. `String` has many uses and I think that's part of why this bug got here.